_includes/configuration/images-upload-handler.md

```diff
@@ -2,9 +2,16 @@
 
 The **images_upload_handler** option allows you to specify a function that is used to replace {{site.productname}}'s default JavaScript upload handler function with custom logic.
 
-The upload handler function takes three arguments: `blobInfo`, a `success` callback, and a `failure` callback. When this option is not set, {{site.productname}} utilizes an `XMLHttpRequest` to upload images one at a time to the server and calls the success callback with the location of the remote image.
+The upload handler function takes four arguments:
 
->Note: Please note that when using this option, no other image uploader options are necessary. Additionally, if you would like {{site.productname}} to replace the `<image>` tag's `src` attribute with the remote location, please use the success callback defined in the `images_upload_handler` function with the returned JSON object's location property.
+* `blobInfo`
+* A `success` callback
+* A `failure` callback
+* A `progress` callback (returning a value between 1 and 100)
```

Comment:
The returning part here is incorrect. It takes a value between 1 and 100 when called, but doesn't return anything.

Reply:
Fixed by: https://github.com/tinymce/tinymce-docs/pull/1565/commits/2d0745a16dbd755d3bdf4cb745c053a8d06110b5